.circleci/verbatim-sources/job-specs/pytorch-job-specs.yml

```diff
@@ -63,6 +63,7 @@ jobs:
             else
               export COMMIT_DOCKER_IMAGE=$output_image
             fi
+            echo "COMMIT_DOCKER_IMAGE: "${COMMIT_DOCKER_IMAGE}
```

Comment:
Same as above
```suggestion
```

Reply:
This one is autogenerated by .circleci/regenerate.sh